pyproject.toml

```diff
@@ -18,7 +18,7 @@ dependencies = [
     "dill",
     "mpi4py",
     "packaging",
-    "radical.pilot >= 1.20"
+    "radical.pilot >= 1.22"
```

Comment:
Do we expect the api_2 branch to be merged for 1.22? Do we expect rp 1.22 to be tagged as a product of this sprint?

Reply:
Yes and yes - but I can't provide a specific release date.